cpp/velox/substrait/SubstraitToVeloxPlan.cc

```diff
@@ -1297,7 +1286,7 @@ int32_t SubstraitToVeloxPlanConverter::getStreamIndex(const ::substrait::ReadRel
     std::string filePath = fileList[0].uri_file();
     std::string prefix = "iterator:";
     std::size_t pos = filePath.find(prefix);
-    if (pos == std::string::npos) {
+    if (pos == std::string::npos || validationMode_) {
```

Comment:
Can we let the function return `-1` fast in `validationMode_` ?

E.g., 
```cpp
if (validationMode_) {
  return -1;
}
```

At the entrance of the function.

cc @rui-mo

Reply:
It should effectively be the same with the patch's code
```cpp
if (pos == std::string::npos || validationMode_) {
  return -1;
}
```

The code looks to return -1 when `validationMode_ == true` anyway?

Reply:
you are right, thanks!